cmake/FindR.cmake

```diff
@@ -109,15 +109,13 @@ find_program(PKG_CONFIG_EXECUTABLE NAMES pkg-config)
                     OUTPUT_STRIP_TRAILING_WHITESPACE)
     set(R_LDFLAGS ${R_LDFLAGS} CACHE PATH "R CMD config --ldflags")
 
-    find_path(R_INCLUDE_DIR R.h
-              HINTS ${R_ROOT_DIR} ${R_ROOT_DIR}/bin/${R_LIB_ARCH}
-              PATHS /usr/local/lib /usr/local/lib64 /usr/share
-              PATH_SUFFIXES include R/include
-              DOC "Path to file R.h")
+    set(R_INCLUDE_DIR "${R_HOME}/include" CACHE PATH "Path to R include directory")
+    
+    # find_library(R_LIBRARY_BASE NAMES R
+    #             HINTS ${R_ROOT_DIR}/lib/R/lib ${R_ROOT_DIR}/bin/${R_LIB_ARCH}
+    #             DOC "R library (example libR.a, libR.dylib, etc.).")
 
-    find_library(R_LIBRARY_BASE R
-                 HINTS ${R_ROOT_DIR}/lib ${R_ROOT_DIR}/bin/${R_LIB_ARCH}
-                 DOC "R library (example libR.a, libR.dylib, etc.).")
+    set(R_LIBRARY_BASE "${R_HOME}/lib/libR.dylib" CACHE PATH "R library (example libR.a, libR.dylib, etc.).")
```

Comment:
Locally on my machine, the `find_library(R_LIBRARY_BASE)` kept finding `/Library/Frameworks/R.framework` instead of the local from the environment, which caused all sorts of trouble. 

Now I realise that this is probably not the right way to go about it, but at least I can work locally on making changes with `hera` etc ...

Reply:
Maybe this should be a separate PR just to set the `R_LIBRARY_BASE` correctly. Perhaps someone more cmake aware can help ?

Reply:
#131 does it better